### DISCUSSION TITLE
What are the suffix options and where can they be used?

### DISCUSSION THREAD

**Discussion by jddcef** [libvips/libvips#3697] [Help wanted]
I want to do something like `vips arrayjoin "file1.tiff file2.tiff" output.png` 
But I want to control the format it is saved in, what are the options I can tweak, e.g. compression level, bit depth etc
I would like it to be a low bit depth to make the file smaller for example.

**Comment by jcupitt** [comment]
Hi @jddcef,

You need to look up the saver. So for PNG it's:

https://www.libvips.org/API/current/VipsForeignSave.html#vips-pngsave
